Architex Axis Suite/src/components/user/UserProfileModal.test.tsx

```
@@ -0,0 +1,252 @@
+
+    await waitFor(() => {
+      expect(screen.queryByText('CAD')).not.toBeInTheDocument();
+    });
+    expect(screen.getByText('Sketching')).toBeInTheDocument();
+    expect(screen.getByText('Project Management')).toBeInTheDocument();
+
+
+    fireEvent.click(screen.getByText('Save Changes'));
+    expect(mockOnSave).toHaveBeenCalledWith(
+      expect.objectContaining({
+        skills: expect.not.arrayContaining(['CAD']),
```

Biome (1.9.4)

[error] This property value named skills is later overwritten by an object member with the same name.

Overwritten with this value.

If an object property with the same name is defined multiple times (except when combining a getter with a setter), only the last definition makes it into the object and previous definitions are ignored.
Unsafe fix: Remove this property value named skills


(lint/suspicious/noDuplicateObjectKeys)